pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -15,6 +24,19 @@ import (
 //       200: ProvisionedAlertRule
 //       404: description: Not found.
 
+// swagger:route GET /api/v1/provisioning/alert-rules/{UID}/export provisioning stable RouteGetAlertRuleExport
```

Comment:
We might be able to drop this endpoint. File provisioning only knows how to read one format, and it requires you to provide entire rule groups. There isn't anything that knows how to read the .yaml produced by this endpoint, it'd be a fragment. If you fed it back into file provisioning as-is, it would be rejected.

Same might also go for the GET `.../alert-rules/export` endpoint - it returns a fragment of the rules, but not the group info.

Reply:
Fair point, I think I might prefer to embed it in the full file structure (return `AlertRuleFileExport`) instead of removing the endpoint entirely but I'm ok either way.